test/test_functional_tensor.py

```diff
@@ -146,25 +148,36 @@ def test_adjustments(self):
                 img = torch.randint(0, 256, shape, dtype=torch.uint8)
 
             factor = 3 * torch.rand(1)
+            hue_factor = torch.add(torch.rand(1), -0.5)
             img_clone = img.clone()
-            for f, ft, sft in fns:
-
-                ft_img = ft(img, factor)
-                sft_img = sft(img, factor)
-                if not img.dtype.is_floating_point:
-                    ft_img = ft_img.to(torch.float) / 255
-                    sft_img = sft_img.to(torch.float) / 255
-
-                img_pil = transforms.ToPILImage()(img)
-                f_img_pil = f(img_pil, factor)
-                f_img = transforms.ToTensor()(f_img_pil)
+            for i, (f, ft, sft) in enumerate(fns):
```

Comment:
Maybe it would be more simple to check the type of `f` like `f == F.adjust_hue` and redefine factor as `factor = torch.rand(1).item() - 0.5` instead of doing `if-else` with almost the same code for both cases...